src/components/Map/map.js

```diff
@@ -108,8 +108,11 @@ class Map extends Component {
         type: "symbol",
         layout: {
           "icon-image": typeId + "icon",
-          "icon-size": 0.4,
-          visibility: "visible"
+          "icon-size": 0.3,
+          "icon-allow-overlap": true, 
+          "icon-ignore-placement": true,
+          visibility: "visible",
```

Comment:
Visibility isn't in quotes, but the other ones are. We should stay consistent. I would prefer no quotes

Reply:
Ooo shit like this wouldn't pass the linter. Point: linter.